### PR TITLE
all: Fix grammatical mistakes in `*.tmpl` files

### PR DESCRIPTION
**Pull request by shmsr**
- Bug
- Docs

## What does this PR do?

It fixes some grammatical mistakes in the *.tmpl files. 

## Why is it important?

It is important because it is important to have solid documentation with as few errors as possible. The template files are sometimes used for configuration and sometimes even for generating code. It's important to have as fewer grammatical mistakes as possible. There's a linked issue as well.

## Checklist

- [x] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## Related issues
- Closes https://github.com/elastic/beats/issues/35696

### CODE REVIEW

File: auditbeat/auditbeat.reference.yml [hunk 1]
```diff
@@ -1473,25 +1473,25 @@ logging.files:
   # The name of the files where the logs are written to.
   #name: auditbeat
 
-  # Configure log file size limit. If limit is reached, log file will be
-  # automatically rotated
+  # Configure log file size limit. If the limit is reached, log file will be
```
Comment (kilfoyle) [inline auditbeat/auditbeat.reference.yml:1476]:
```suggestion
  # Configure log file size limit. If the limit is reached, the log file will be
```

File: auditbeat/auditbeat.reference.yml [hunk 2]
```diff
@@ -1473,25 +1473,25 @@ logging.files:
...
+  # automatically rotated.
```
Comment (kilfoyle) [inline auditbeat/auditbeat.reference.yml:1477]:
```suggestion
  # Configure log file size limit. If the limit is reached, the log file will be
  # automatically rotated.
```

File: filebeat/_meta/config/filebeat.inputs.reference.yml.tmpl [hunk 3]
```diff
@@ -280,7 +280,7 @@ filebeat.inputs:
 
   ### Prospector options
 
-  # How often the input checks for new files in the paths that are specified
+  # How often do the input checks for new files in the paths that are specified
```
Comment (kilfoyle) [inline filebeat/_meta/config/filebeat.inputs.reference.yml.tmpl:283]:
```suggestion
  # How often the input checks for new files in the paths that are specified
```
I think the original is correct in this case.

File: filebeat/_meta/config/filebeat.inputs.reference.yml.tmpl [hunk 4]
```diff
@@ -526,7 +526,7 @@ filebeat.inputs:
   # List of hosts to pool to retrieve the slow log information.
   #hosts: ["localhost:6379"]
 
-  # How often the input checks for redis slow log.
+  # How often do the input checks for redis slow log.
```
Comment (kilfoyle) [inline filebeat/_meta/config/filebeat.inputs.reference.yml.tmpl:529]:
```suggestion
  # How often the input checks for redis slow log.
```
I think the original is correct as is.

File: filebeat/filebeat.reference.yml [hunk 5]
```diff
@@ -933,7 +933,7 @@ filebeat.inputs:
   # List of hosts to pool to retrieve the slow log information.
   #hosts: ["localhost:6379"]
 
-  # How often the input checks for redis slow log.
+  # How often do the input checks for redis slow log.
```
Comment (kilfoyle) [inline filebeat/filebeat.reference.yml:936]:
```suggestion
  # How often the input checks for redis slow log.
```
I think the original is okay as is.

File: heartbeat/_meta/config/beat.reference.yml.tmpl [hunk 6]
```diff
@@ -10,7 +10,7 @@
 ############################# Heartbeat ######################################
 
 
-# Define a directory to load monitor definitions from. Definitions take the form
+# Define a directory to load monitor definitions. Definitions take the form
```
Comment (kilfoyle) [inline heartbeat/_meta/config/beat.reference.yml.tmpl:13]:
```suggestion
# Define a directory from which to load monitor definitions. Definitions take the form
```

File: heartbeat/_meta/config/beat.reference.yml.tmpl [hunk 7]
```diff
@@ -159,8 +159,8 @@ heartbeat.monitors:
   # Set to true to publish fields with null values in events.
   #keep_null: false
 
-- type: http # monitor type `http`. Connect via HTTP an optionally verify response
-  # ID used to uniquely identify this monitor in elasticsearch even if the config changes
+- type: http # monitor type `http`. Connect via HTTP an optionally verify the response
+  # ID used to uniquely identify this monitor in Elasticsearch even if the config changes
```
Comment (kilfoyle) [inline heartbeat/_meta/config/beat.reference.yml.tmpl:163]:
```suggestion
- type: http # monitor type `http`. Connect via HTTP and optionally verify the response
  # ID used to uniquely identify this monitor in Elasticsearch even if the config changes.
```

File: heartbeat/_meta/config/beat.yml.tmpl [hunk 8]
```diff
@@ -9,7 +9,7 @@
 
 ############################# Heartbeat ######################################
 
-# Define a directory to load monitor definitions from. Definitions take the form
+# Define a directory to load monitor definitions. Definitions take the form
```
Comment (kilfoyle) [inline heartbeat/_meta/config/beat.yml.tmpl:12]:
```suggestion
# Define a directory from which to load monitor definitions. Definitions take the form
```

File: heartbeat/heartbeat.reference.yml [hunk 9]
```diff
@@ -10,7 +10,7 @@
 ############################# Heartbeat ######################################
 
 
-# Define a directory to load monitor definitions from. Definitions take the form
+# Define a directory to load monitor definitions. Definitions take the form
```
Comment (kilfoyle) [inline heartbeat/heartbeat.reference.yml:13]:
```suggestion
# Define a directory from which to load monitor definitions. Definitions take the form
```

File: heartbeat/heartbeat.yml [hunk 10]
```diff
@@ -9,7 +9,7 @@
 
 ############################# Heartbeat ######################################
 
-# Define a directory to load monitor definitions from. Definitions take the form
+# Define a directory to load monitor definitions. Definitions take the form
```
Comment (kilfoyle) [inline heartbeat/heartbeat.yml:12]:
```suggestion
# Define a directory from which to load monitor definitions. Definitions take the form
```

File: x-pack/heartbeat/heartbeat.reference.yml [hunk 11]
```diff
@@ -10,7 +10,7 @@
 ############################# Heartbeat ######################################
 
 
-# Define a directory to load monitor definitions from. Definitions take the form
+# Define a directory to load monitor definitions. Definitions take the form
```
Comment (kilfoyle) [inline x-pack/heartbeat/heartbeat.reference.yml:13]:
```suggestion
# Define a directory from which to load monitor definitions. Definitions take the form
```

File: x-pack/heartbeat/heartbeat.reference.yml [hunk 12]
```diff
@@ -1565,25 +1565,25 @@ logging.files:
   # The name of the files where the logs are written to.
   #name: heartbeat
 
-  # Configure log file size limit. If limit is reached, log file will be
-  # automatically rotated
+  # Configure log file size limit. If the limit is reached, log file will be
+  # automatically rotated.
```
Comment (kilfoyle) [inline x-pack/heartbeat/heartbeat.reference.yml:1569]:
```suggestion
  # Configure log file size limit. If the limit is reached, the log file will be
  # automatically rotated.
```

File: x-pack/metricbeat/metricbeat.reference.yml [hunk 13]
```diff
@@ -2853,25 +2853,25 @@ logging.files:
   # The name of the files where the logs are written to.
   #name: metricbeat
 
-  # Configure log file size limit. If limit is reached, log file will be
-  # automatically rotated
+  # Configure log file size limit. If the limit is reached, log file will be
+  # automatically rotated.
```
Comment (kilfoyle) [inline x-pack/metricbeat/metricbeat.reference.yml:2857]:
```suggestion
  # Configure log file size limit. If the limit is reached, the log file will be
  # automatically rotated.
```

File: x-pack/packetbeat/packetbeat.reference.yml [hunk 14]
```diff
@@ -162,15 +162,15 @@ packetbeat.protocols:
   #send_request: true
 
   # If this option is enabled, the raw message of the response (`cassandra_request.request_headers` field)
-  # is included in published events. The default is true. enable `send_request` first before enable this option.
+  # is included in published events. The default is true. enable `send_request` first before enabling this option.
```
Comment (kilfoyle) [inline x-pack/packetbeat/packetbeat.reference.yml:165]:
```suggestion
  # is included in published events. The default is true. Enable `send_request` first before enabling this option.
```

File: x-pack/packetbeat/packetbeat.reference.yml [hunk 15]
```diff
@@ -162,15 +162,15 @@ packetbeat.protocols:
...
   #send_request_header: true
 
   # If this option is enabled, the raw message of the response (`cassandra_response` field)
   # is included in published events. The default is true.
   #send_response: true
 
   # If this option is enabled, the raw message of the response (`cassandra_response.response_headers` field)
-  # is included in published events. The default is true. enable `send_response` first before enable this option.
+  # is included in published events. The default is true. enable `send_response` first before enabling this option.
```
Comment (kilfoyle) [inline x-pack/packetbeat/packetbeat.reference.yml:173]:
```suggestion
  # is included in published events. The default is true. Enable `send_response` first before enabling this option.
```

File: x-pack/packetbeat/packetbeat.reference.yml [hunk 16]
```diff
@@ -1939,25 +1939,25 @@ logging.files:
   # The name of the files where the logs are written to.
   #name: packetbeat
 
-  # Configure log file size limit. If limit is reached, log file will be
-  # automatically rotated
+  # Configure log file size limit. If the limit is reached, log file will be
+  # automatically rotated.
```
Comment (kilfoyle) [inline x-pack/packetbeat/packetbeat.reference.yml:1943]:
```suggestion
  # Configure log file size limit. If the limit is reached, the log file will be
  # automatically rotated.
```

**Review by kilfoyle — APPROVED** [review]
LGTM for the text changes! 🚀 
Hopefully one of the Beats code owners can assist with the CI checks and merging.

**Comment by shmsr** [comment]
> LGTM for the text changes! 🚀 Hopefully one of the Beats code owners can assist with the CI checks and merging, since these changes are in code files.

Thanks @kilfoyle for reviewing.

**Review by belimawr — APPROVED** [review]

**Comment by shmsr** [comment]
@belimawr Whom should I ask for review from `elastic/{uptime,security-external-intergrations,security-assent-management}`?

File: filebeat/_meta/config/filebeat.global.reference.yml.tmpl [hunk 17]
```diff
@@ -4,14 +4,14 @@
...
+# The permissions mask to apply on registry data and meta files. The default
 # value is 0600.  Must be a valid Unix-style file permissions mask expressed in
 # octal notation.  This option is not supported on Windows.
 #filebeat.registry.file_permissions: 0600
 
-# The timeout value that controls when registry entries are written to disk
+# The timeout value that controls when registry entries are written to the disk
 # (flushed). When an unwritten update exceeds this value, it triggers a write
-# to disk. When flush is set to 0s, the registry is written to disk after each
+# to disk. When the flush is set to 0s, the registry is written to disk after each
```
Comment (andrewkroh) [inline filebeat/_meta/config/filebeat.global.reference.yml.tmpl:14]: `flush` in this context is being used as the variable name. Making it "the flush" sounds worse IMO. I would either leave this as is or change it to "When the value is set to 0s".

**Review by andrewkroh — APPROVED** [review]

File: filebeat/_meta/config/filebeat.global.reference.yml.tmpl (hunk 17 above)
Comment (shmsr) [reply filebeat/_meta/config/filebeat.global.reference.yml.tmpl:14]: Okay, I'll do it.

Comment (shmsr) [reply filebeat/_meta/config/filebeat.global.reference.yml.tmpl]: Done!

**Comment by shmsr** [comment]
Ping @elastic/security-asset-management @elastic/uptime. Could someone please review this PR?

**Review by andrewvc — APPROVED** [review]
LGTM for the heartbeat portion

**Review by aleksmaus — APPROVED** [review]
osquerybeat updates LGTM, thank you!

**Pull request merged**